packages/browser/rollup.config.mjs

```diff
@@ -0,0 +1,39 @@
+// The following is only possible from Node 18 onwards
+// import pkg from "./package.json" assert { type: "json" };
+
+// Until we only support Node 18+, this should be used instead
+// (see https://rollupjs.org/guide/en/#importing-packagejson)
+import { createRequire } from "node:module";
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import terser from '@rollup/plugin-terser';
+import createConfig, { sharedConfig } from '../../rollup.common.mjs';
+
+const require = createRequire(import.meta.url);
+const pkg = require("./package.json");
+
+export default [
+  ...createConfig(pkg),
+  {
+    input: "./src/index.ts",
+    output: [
+      {
+        file: pkg.bundle,
+        format: "iife",
```

Comment:
Ah good, that's non-breaking for the scrip tags imports ? if we can dump webpack and focus on rollup, that's great.

Reply:
Can confirm that https://github.com/inrupt/solid-client-authn-js/tree/main/packages/browser/examples/single/script works. Would it be worth adding a playwright test at some point for that one?

Reply:
Absolutely, can you create a ticket to track this?